packages/schema/package.json

```
@@ -27,7 +27,7 @@
   "scripts": {
     "build": "jlpm build:schema && jlpm build:lib",
     "build:schema": "jlpm build:schema:js && jlpm build:schema:py",
```

Shouldn't it be this instead? Downloading and caching geojson schema before generating interfaces for TS and Python?

```suggestion
    "build:schema": "node ./cacheGeoJSONSchema.js && jlpm build:schema:js && jlpm build:schema:py",
```

---

Ahh yes, i didn't realize